requirements.txt

```diff
@@ -17,6 +19,7 @@ django-bootstrap3==10.0.1
 django-compressor==2.2
 django-cors-headers==2.2.0
 django-crequest==2018.5.11
+django-environ==0.4.5
 django-filter==2.0.0
 django-helusers==0.4.2
 django-ipware==2.1.0
```

Comment:
Any chance you can change `django-translation-checker` and `python3-saml` dependency from using both `https+git` instead of `git+git` only.

I have problem setting up tunnistamo even i got ssh but inside Hki network.

Reply:
That we can do, good point!

Reply:
Fixed